Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "apialerts"
-version = "0.1.0"
+version = "2.0.0"
```

Comment:
Lets jump to 1.0.0 rather than 2. I'll push a few other standardising updates along with the 2.0 release for all sdks

Reply:
🤦  doh I forgot

Reply:
done